runtime/moonbase/tests/xcm_mock/mod.rs

```diff
@@ -150,6 +151,29 @@ pub fn para_ext(para_id: u32) -> sp_io::TestExternalities {
 	.assimilate_storage(&mut t)
 	.unwrap();
 
+	frame_support::traits::GenesisBuild::<Runtime>::assimilate_storage(
+		&pallet_xcm_transactor::GenesisConfig {
+			// match relay runtime construct_runtime order in xcm_mock::relay_chain
+			relay_indices: RelayChainIndices {
```

Comment:
why not use westend indices here?

Reply:
done 👍

Reply:
This change somehow broke the tests. I have reverted it.